Decisions.TruCap/Steps/DocumentMonitorSteps.cs

```diff
@@ -27,21 +27,21 @@ public class DocumentMonitorSteps
             }
             
             string baseUrl = ModuleSettingsAccessor<TruCapSettings>.GetSettings().GetBaseDocumentMonitorUrl(overrideBaseUrl);
-            Task<HttpResponseMessage> response = TruCapRest.TruCapGet(
+            Task<string> resultTask = TruCapRest.TruCapGet(
                 $"{baseUrl}/Project/{project}/DocumentSubType/{docSubType}/FromDate/{startDate}/ToDate/{endDate}",
                 authentication);
 
             try
             {
-                return DocumentMonitorResponse.JsonDeserialize(await response.Result.Content.ReadAsStringAsync());
+                return DocumentMonitorResponse.JsonDeserialize(resultTask.Result);
```

Comment:
We can't do this either. We have to wait for the task to complete like we've done everywhere.

Reply:
TruCapGet() waits